### DISCUSSION TITLE
Installing FFTW3, NetCDF and BLAS/LAPACK (needed for Surface Topography) on Windows

### DISCUSSION THREAD

**Discussion by sitangshugk95** [ContactEngineering/SurfaceTopography#221] [Q&A]
I'm trying to run the package on Windows, according to the steps mentioned in https://github.com/ContactEngineering/SurfaceTopography/blob/0edd969fe922710cdc0fae22a3a73b142d88a3ed/docs/installation.rst ; however, the steps for installing FFTW3, NetCDF and BLAS/LAPACK are only given for Ubuntu and Linux. Can someone please provide me with a similar command that can be used for Windows?
![pip install](https://user-images.githubusercontent.com/108830413/183828923-c37c0e48-6d37-4394-8bdc-fc34e96929c5.jpg)

**Comment by sannant** [comment]
We do not support installation on Windows. I update the documentation to make this more explicit in this PR https://github.com/ContactEngineering/SurfaceTopography/pull/220. 

Sorry for the inconvenience.

**Comment by pastewka** [comment]
If you want to run on Windows, I recommend using the [Windows subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install).

**Reply by sitangshugk95** [reply]
I have gotten access to a MAC, and will soon get access to Linux too. I'm waiting until you get a chance to look into the issue with the muFFT. Is there any update on that?

**Reply by pastewka** [reply]
WSL is probably easier to setup than on macOS
